src/run.py

```diff
@@ -46,7 +46,7 @@ def main():
             "def4",
             "def5",
             "def6",
-            "def7", # FP - Very similar to kown issue
+            "def7", # FP - Very similar to known issue
```

Comment:
Can you also rename the `src/model` to `src/dto`. Because model and imply AI models since this project also involves AI models.